<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Used address</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Pool</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ownerField">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="addressField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Owner</string>
    </property>
   </widget>
   <widget class="QListWidget" name="poolField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="infoField">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
